docs/README.ja.md

```diff
@@ -1 +1,102 @@
-TODO
\ No newline at end of file
+<p align="center"><img src="/src/icons/icon@128.png" alt="holo-schedule logo"></p>
+
+# holo-schedule
+
+![Test status](https://github.com/YunzheZJU/holo-schedule/workflows/Test/badge.svg)
+
+—— Keep in touch with Hololive.
+
+ホロライブの配信を表示するブラウザ拡張です。
+
+[Eng](/README.md) [中文](docs/README.zh_CN.md)
+
+### 目次
+
+1. [インストール](#インストール)
+1. [対応ブラウザ](#対応ブラウザ)
+1. [依存関係](#依存関係)
+1. [ビルド](#ビルド)
+1. [テスト](#テスト)
+1. [開発](#開発)
+
+## インストール
+
+以下の公式ページでダウンロードできます。
+[Mozilla AMO](https://addons.mozilla.org/firefox/addon/holo-schedule/) 
+または  
```

Comment:
A doubled space will lead to an unexpected new line. Ignore this suggestion if this new line is intended.
```suggestion
または
```